.eslintrc.js

```diff
@@ -5,8 +5,10 @@ module.exports = {
   },
   extends: [
     'plugin:vue/essential',
+    // 'plugin:vue/recommended',
     '@vue/standard',
     '@vue/typescript'
+    // '@vue/typescript/recommended'
```

Comment:
I'm thinking lines 9 and 11 are worthy changes but I'm trying to limit the impact of the upgrade -- we can use recommended lint rules, and fix what it finds, later.